.azure_pipeline.yml

```diff
@@ -81,7 +81,7 @@ stages:
           VERSION_PYTHON: '3.8'
           CC_OUTER_LOOP: 'gcc'
           CC_INNER_LOOP: 'clang-10'
-        # Linux environment with numpy from conda-forge channel
+        # Linux environment with numpy from conda-forge channel and openblas-openmp
```

Comment:
I think the configuration needs to be updated to enforce the openmp threading layer for openblas  because the default is platform specific and might depend on the version of the package.